charts/fluent-bit/templates/ingress.yaml

```diff
@@ -14,7 +14,9 @@ metadata:
     {{- include "fluent-bit.labels" . | nindent 4 }}
   {{- with .Values.ingress.annotations }}
   annotations:
-    {{- toYaml . | nindent 4 }}
+    {{- range $key, $value := . }}
+      {{- printf "%s: %s" $key ((tpl $value $) | quote) | nindent 4 }}
+    {{- end }}
```

Comment:
```suggestion
  {{- range $key, $value := . }}
    {{ printf "%s: %s" $key ((tpl $value $) | quote) }}
  {{- end }}
```
The indentation you've used is incorrect, although would work due to the unnecessary `nindent`.